snapcraft/wrappers/lxd-stophook

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+# Use exec so that this script process is replaced.
+# This avoids polluting the process tree with this wrapper script.
+if [ "$1" = "callhook" ]; then
+    exec /snap/lxd/current/bin/lxd-user "$@"
+else
```

Comment:
`s/else/fi/` :)

Reply:
I fixed this but must not have pushed it after hitting the issue with the snapd unconfined rule changes